src/app/Core/ExportServices/dataOrStatisticToCsvString.service.ts

```diff
@@ -21,10 +21,13 @@ export interface ObjectToCsvParameter {
  * @param {ExportInterface} setting - all info for export (raw/stat, row&column, dataset)
  * @param {FilterInterface} filter - object with the selected filters
  * @param {string} allFilterLabel - "all values" / "Alle Werte"
- * @param {MainFilterLabelInterface} mainFilterLabels - object with labels of the main filters
+ * @param {MainFilterLabelInterface} mainFilterLabels - Obj with translated labels for the table header
+ * @param {string[]} - keys to get matching mainFilterLabel
```

Comment:
this is the description for the ObjectToCsvParameter interface, the function only has one "csvParameter"

Reply:
Done. Commit "Simplify code"